app/controllers/integrations/feature_flags_controller.rb

```diff
@@ -10,6 +10,11 @@ def index
         features
       end
 
+      feature_flags['sandbox_mode'] = {
+        name: 'Sandbox mode',
+        active: HostingEnvironment.sandbox_mode?,
+      }
```

Comment:
This will make the screen report the feature flag as wrong. Can we put it in another key and display it separately?